app/v1/controllers/requestParsers/validators/AmendEmploymentExpensesValidator.scala

```diff
@@ -46,6 +46,11 @@ class AmendEmploymentExpensesValidator @Inject()(implicit currentDateTime: Curre
     )
   }
 
+  private def incorrectOrEmptyBodySubmittedValidation: AmendEmploymentExpensesRawData => List[List[MtdError]] = { data =>
```

Comment:
can the validation be included in the `bodyFormatValidation` in the `JsonFormatValidation`

Reply:
I believe the validation can't be in the same def as `bodyFormatValidation` since it needs to check that it is a valid json before it tries to check if the object is empty. Otherwise currently when I try the tests give JsResultException Errors.